mathics_django/web/media/js/graphics3d.js

```diff
@@ -2,26 +2,28 @@ const drawFunctions = {
 	point: (element) => {
 		const geometry = new THREE.Geometry();
 
-		element.coords.forEach((coordinate) => {
-			const point = new THREE.Mesh(
-				new THREE.SphereGeometry(element.pointSize / 2, 24, 24)
-			);
-
-			point.position.set(...coordinate[0]);
-
-			point.updateMatrix();
-
-			geometry.merge(point.geometry, point.matrix);
-		});
-
-		geometry.computeFaceNormals();
-
-		return new THREE.Mesh(
-			geometry,
-			new THREE.MeshBasicMaterial({
-				color: new THREE.Color().setRGB(...element.color).getHex()
-			})
+		geometry.vertices = element.coords.map(
+			(coordinate) => new THREE.Vector3(...coordinate[0])
 		);
+
+		return new THREE.Points(geometry, new THREE.ShaderMaterial({
+			transparent: true,
+			uniforms: {
+				// 84 is the scale for this size of canvas
```

Comment:
I was playing around with this value today with: 

```
Graphics3D[Table[Point[{x, y, z}], {x, 5}, {y, 5}, {z, 5}]]
```

and 200 seems to be a reasonable number. 200 is also used in toCanvasCoords() on line 552. 

Ideally though the magic number 200 would be set as a global constant. Or even better it is probably should be adjustable and set as an attribute inside some object either global, or found in one of the passed-in parameters.

Reply:
@mmatera [here](https://github.com/Mathics3/mathics-django/issues/66) has the fix of the size of the canvas (it's relative size, I just need to add a multiplier of 0.5, I didn't see the comment of @rocky before).

Reply:
> Hi,
> I was trying to follow this discussion, and maybe I haven't done it completely.

It goes without saying (but I will say it anyway), that it is always good to hear from you!

> 
> ...
> The second issue maybe could be fixed if we can pass a relative size for the dot size instead of an absolute one. Would be that possible in three.js?

This was done in https://github.com/mathics/Mathics/pull/1411

and I renamed the less helpful name `PointSize.get_size()` to `PointSize.get_absolute_size()` later on.

In https://github.com/mathics/Mathics/pull/1415 I tried changing asymptote to use circles for points which are adjustable in the circle diameter while the dot size as far as I know isn't. 

However there is something incomplete and this didn't pass CircleCI's testing in using asymptote to build the doc. So I have these changes in comments and at some point we will come back to it.

In general though, I think using a relative size in the specification of PointSize and leaving at that for the formatter is the right way to go, and the extra code and complexity which was in the code base right now is not helpful.